packages/Toast/README.md

```diff
@@ -25,10 +25,10 @@ npm install @paprika/toast
 | Prop           | Type                                                                                                                                                            | required | default               | Description                                                                                                                         |
 | -------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- | --------------------- | ----------------------------------------------------------------------------------------------------------------------------------- |
 | autoCloseDelay | number                                                                                                                                                          | false    | 5000                  | Duration (in ms) before Toast will automatically close (if canAutoClose is true).                                                   |
-| canAutoClose   | bool                                                                                                                                                            | false    | false                 | Will automatically close after 1500ms (or longer if provided by autoCloseDelay).                                                    |
+| canAutoClose   | bool                                                                                                                                                            | false    | false                 | Will automatically close after 5000ms (or longer if provided by autoCloseDelay).                                                    |
```

Comment:
is 5000ms a new value? was this suggested by a ux-design :) how did we arrive at this value?

Reply:
The default value for autoCloseDelay was always 5000ms, so technically the default behaviour was that, so I updated the docs to reflect that. However, there _is_ a 'minimumCloseTime' of 1500ms.

Right now the timeout timer is set by Math.max(autoCloseDelay, minimumCloseTime).